### PR TITLE
ui: get relative time from user for history

### PR DESCRIPTION
**Pull request by masco-enovance**

### CODE REVIEW

File: statics/js/components/topology.js [hunk 1]
```diff
@@ -521,10 +554,24 @@ var TopologyComponent = {
...
+      var h = relTimeStr.match(/(\d+)\s*h/);
+      var m = relTimeStr.match(/(\d+)\s*m/);
+      var s = relTimeStr.match(/(\d+)\s*s/);
+      var totalTime = 0;
+      if (d) totalTime += parseInt(d[1]) * 86400;
+      if (h) totalTime += parseInt(h[1]) * 3600;
+      if (m) totalTime += parseInt(m[1]) * 60;
+      if (s) totalTime += parseInt(s[1]);
+      totalTime = totalTime * 1000;
+      currentTime = new Date().getTime();
```
Comment (safchain) [inline statics/js/components/topology.js]: var missing

File: statics/js/components/topology.js [hunk 2]
```diff
@@ -521,10 +554,24 @@ var TopologyComponent = {
...
+      absTime = new Date(currentTime - totalTime);
```
Comment (safchain) [inline statics/js/components/topology.js]: same here

File: statics/js/components/topology.js (hunk 1 above)
Comment (masco-enovance) [reply statics/js/components/topology.js]: done

File: statics/js/components/topology.js (hunk 2 above)
Comment (masco-enovance) [reply statics/js/components/topology.js]: done

**Pull request merged**